thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -175,46 +176,67 @@ export function evalObj() {
   };
 }
 
+/**
+ * Builds the request parameters for the metric data API call.
+ * @method buildMetricDataUrl
+ * @param {Object} graphConfig - the metric settings
+ * @returns {String} metric data call params/url
+ */
+export function buildMetricDataUrl(graphConfig) {
+  const { id, maxTime, filters, dimension, granularity } = graphConfig;
+  const selectedDimension = dimension ? dimension : 'All';
```

Comment:
shorter way of writing this: 
`const selectedDimension = dimension || 'All'`

Reply:
or using `Ember. getWithDefault(graphConfig, 'dimension', 'All')`
^ just fyi, this might results in more line of codes in our case

Reply:
sure - thanks.